client/src/modules/Auth/SignIn/index.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import ModalWrapper from '../../../shared/ModalWrapper';
 import SignInModal from './components/SignInModal';
 import ButtonSignIn from './components/ButtonSignIn';
 import ButtonUser from './components/ButtonUser';
 
-const auth = false;
-
-const SignIn = () => {
+const SignIn = ({ user }) => {
   return (
     <>
-      {auth ? (
+      {user ? (
         <ButtonUser />
       ) : (
         <ModalWrapper ButtonOpen={ButtonSignIn} ModalContent={SignInModal} />
```

Comment:
User is object, so even empty object will return "true" and You never get ModalWrapper component.